File: hp-server/src/main/java/net/hserver/hplite/domian/entity/UserStatisticsEntity.java
package net.hserver.hplite.domian.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

/**
 * 用户的套餐
 */
@Data
@TableName("user_statistics")
@NoArgsConstructor
public class UserStatisticsEntity {

    @TableId(type = IdType.AUTO)
    private Integer id;

    /**
     * 套餐ID
     */
    private Integer configId;

    /**
     * 下载量
     */
    private Long download;

    /**
     * 上传量
     */
    private Long upload;

    /**
     * uv
     */
    private Integer uv;

    /**
     * pv
     */
    private Integer pv;

    /**
     * 时间
     */
    private Long time;

    /**
     * 创建时间
     */
    private Date createTime;

}

File: hp-server-golang/hp-server-lib/entity/user_statistics_entity.go
package entity

import "time"

type UserStatisticsEntity struct {
	Id *int `json:"id" gorm:"primaryKey;autoIncrement"`

	/**
	 * 套餐ID
	 */
	ConfigId int `json:"configId"`

	/**
	 * 下载量
	 */
	Download int64 `json:"download"`

	/**
	 * 上传量
	 */
	Upload int64 `json:"upload"`

	/**
	 * uv
	 */
	Uv int64 `json:"uv"`

	/**
	 * pv
	 */
	Pv int64 `json:"pv"`

	/**
	 * 时间
	 */
	Time int64 `json:"time"`

	/**
	 * 创建时间
	 */
	CreateTime time.Time `json:"createTime"`
}

func (UserStatisticsEntity) TableName() string {
	return "user_statistics"
}
